src/Entity/SmsMessage.php

```diff
@@ -335,6 +337,23 @@ public function setProcessedTime($processed) {
   /**
    * {@inheritdoc}
    */
+  public function chunkByRecipients($size) {
```

Comment:
There are use cases where one may want to send large messages without storage, for example where the SMS framework is acting as an API passing message along.
I suggest we should move this method into the Message\SmsMessage class and interface (and perhaps have another implementation here if still needed).